example/package.json

```diff
@@ -15,7 +15,7 @@
     "eject": "node ../node_modules/react-scripts/bin/react-scripts.js eject"
   },
   "dependencies": {
-    "@EasyDynamics/oscal-react-library": "file:..",
+    "@easydynamics/oscal-react-library": "file:..",
```

Comment:
Is there a reason we switched from `EasyDynamics --> easydynamics`? 

I am wondering if that is what is causing the error:
```
Module not found: Error: Can't resolve '@easydynamics/oscal-react-library' in '/home/runner/work/oscal-react-library/oscal-react-library/example/src'
```

Reply:
That was done as part of publishing to `npm`; we have to be all-lowercase. Make sure to `npm ci` in both directories.